client/src/app/pages/issues/issue-detail-drawer/file-incidents-detail-modal/file-incidents-detail-modal.tsx

```diff
@@ -58,7 +58,10 @@ export const FileIncidentsDetailModal: React.FC<
     isFetching ||
     (firstFiveIncidents.length > 0 && activeTabIncidentId === undefined);
 
-  const issueTitle = issue.description.split("\n")[0];
+  const issueTitle =
+    issue.description.split("\n")[0] ||
+    issue.name.split("\n")[0] ||
+    "Issue name";
```

Comment:
Seems like something other than "issue name" would be better.
perhaps:
- unnamed
- anonymous
- not-named